<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CRSIISClass</class>
 <widget class="QMainWindow" name="CRSIISClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CRSIIS</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>685</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Raster"/>
    <addaction name="actionOpen_Vector"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionROI"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen_Raster"/>
   <addaction name="actionOpen_Vector"/>
   <addaction name="actionROI"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionOpen_Raster">
   <property name="text">
    <string>Open Raster</string>
   </property>
  </action>
  <action name="actionOpen_Vector">
   <property name="text">
    <string>Open Vector</string>
   </property>
  </action>
  <action name="actionROI">
   <property name="text">
    <string>ROI</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="crsiis.qrc"/>
 </resources>
 <connections/>
</ui>
